bot/pagination.py

```diff
@@ -229,20 +230,37 @@ def event_check(reaction_: discord.Reaction, user_: discord.Member) -> bool:
                 or isinstance(user_, Member) and any(role.id in MODERATION_ROLES for role in user_.roles)
             )
 
-            return (
+            right_reaction = (
+                # Reaction is on this message
+                reaction_.message.id == message.id
+                # Reaction is one of the pagination emotes
+                and str(reaction_.emoji) in PAGINATION_EMOJI
+            )
+
+            whitelisted = (
                 # Conditions for a successful pagination:
                 all((
-                    # Reaction is on this message
-                    reaction_.message.id == message.id,
-                    # Reaction is one of the pagination emotes
-                    str(reaction_.emoji) in PAGINATION_EMOJI,
                     # Reaction was not made by the Bot
                     user_.id != ctx.bot.user.id,
+                    # It was the right reaction and Message
+                    right_reaction,
                     # There were no restrictions
                     no_restrictions
                 ))
             )
 
+            if whitelisted:
+                return True
+
+            elif right_reaction and user_.id != ctx.bot.user.id:
+
+                scheduling.create_task(
+                    (discord.HTTPException,),  # Suppress the HTTPException if adding the reaction fails
+                    reaction.message.remove_reaction(reaction.emoji, user)
+                )
```

Comment:
You forgot to change these calls to use the new signature with `*args`.

Reply:
Whoops, I should have tested it more.